MOPTools/twa_mops/om.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+from twa.data_model.base_ontology import BaseOntology, BaseClass, ObjectProperty, DatatypeProperty
+
+
+# NOTE TODO this script is incomplete, and it should be moved to a place that accessible to all other ontologies
```

Comment:
Please briefly indicate what is missing.

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1330/commits/2b36b35b9b6d806f9bbabfc6bebf16ee822f2f51